test/test.js

```diff
@@ -9,41 +9,41 @@ import presetLintNode from "../index.js";
 import { read } from "to-vfile";
 import { reporter } from "vfile-reporter";
 
+import fixturesUrl from "./fixtures/fixturesUrl.js";
+
 const linter = unified()
   .use(remarkParse)
   .use(presetLintNode)
   .use(remarkStringify);
 
 const handleError = (err) => {
   console.error(err);
-  process.exit(1);
+  process.exitCode = 1;
```

Comment:
Do we've got the 3 tests running async using this one. If 1 fails and 2 pass, is the exitCode going to be overridden/race conditioned if it doesn't immediately exit on the first failure?

Reply:
Unless there's a `process.exitCode = 0` statement executed during the tests, it shouldn't be an issue: if there's no failure, the exitCode stay untouched at 0, if there's one or more, it will be assigned to the value `1` (possibly several time, but I don't see why that would be an issue).